modules/tinymce/src/plugins/quickbars/test/ts/browser/KeyboardShortcutContextualMenuTest.ts

```diff
@@ -0,0 +1,33 @@
+import { FocusTools, Keys } from '@ephox/agar';
+import { describe, it } from '@ephox/bedrock-client';
+import { SugarDocument } from '@ephox/sugar';
+import { TinyHooks, TinySelections, TinyContentActions, McEditor, TinyUiActions } from '@ephox/wrap-mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import QuickbarsPlugin from 'tinymce/plugins/quickbars/Plugin';
+
+describe('browser.tinymce.core.focus.KeyboardShortcutContextualMenuTest', () => {
```

Comment:
This name is incorrect, it should match similar to the file path (minus the test/ts/browser part).
```suggestion
describe('browser.tinymce.plugins.quickbars.KeyboardShortcutContextualMenuTest', () => {
```